apps/api/src/app/environments/environment.ts

```diff
@@ -131,6 +131,10 @@ const devConfig = {
     apiKey: process.env.PKPASS_API_KEY,
     apiUrl: process.env.PKPASS_API_URL,
     secretKey: process.env.PKPASS_SECRET_KEY,
+    cacheKey: process.env.PKPASS_CACHE_KEY ?? 'smartsolution:apitoken',
```

Comment:
I worry that `??` might be overly specific. Is there no chance of the env variable being present but empty (`""`)?

Reply:
technically it could happen, but I'm following a pattern in the file and this is not done for prod settings, only as convenience in dev